_topic_maps/_topic_map.yml

```diff
@@ -24,7 +24,7 @@
 ---
```

Comment:
pretty sure this file shouldn't be in the PR :/

Reply:
Yep, you're right. Deleted.